imagefiles/build-and-install-ninja.sh

```diff
@@ -31,25 +37,19 @@ if [[ -z "${NINJA_VERSION}" ]]; then
 fi
 
 # Download
-url="https://github.com/ninja-build/ninja/archive/v${NINJA_VERSION}.tar.gz"
-
-curl --connect-timeout 30 \
-    --max-time 10 \
-    --retry 5 \
-    --retry-delay 10 \
-    --retry-max-time 30 \
-    -# -o ninja.tar.gz -LO "$url"
+NINJA_DOWNLOAD_URL="https://github.com/ninja-build/ninja/archive/"
+NINJA_ROOT="v${NINJA_VERSION}"
 
-mkdir ninja
-tar -xzvf ./ninja.tar.gz --strip-components=1 -C ./ninja
+fetch_source "${NINJA_ROOT}.tar.gz" "${NINJA_DOWNLOAD_URL}"
+tar xvzf "${NINJA_ROOT}.tar.gz"
```

Comment:
Why does it remove the `--strip-components=1` part? Not sure exactly why it's there, but I assume it was there for a reason? 🙈